probe-rs/src/architecture/arm/ap/mod.rs

```diff
@@ -82,12 +79,15 @@ impl AccessPortError {
 /// A trait to be implemented by access port register types.
 ///
 /// Use the [`define_ap_register!`] macro to implement this.
-pub trait ApRegister<PORT: AccessPort>: Register + Sized {}
+//pub trait ApRegister<PORT: AccessPortType>: Register + Sized {}
```

Comment:
I don't think we want to keep dead code around. Do you have a purpose for this still?

Reply:
@ithinuel I guess this got lost, do you want to keep the commented dead code?